content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -399,7 +399,7 @@ rolled back.
 * Press Ctrl-C to stop the above rollout status watch. For more information on stuck rollouts,
 [read more here](#deployment-status).
 
-* You see that the number of old replicas (`nginx-deployment-1564180365` and `nginx-deployment-2035384211`) is 2, and new replicas (nginx-deployment-3066724191) is 1.
+* You see that the number of old replicas (`nginx-deployment-1564180365`) is 3, and new replicas (`nginx-deployment-3066724191`) is 1.
```

Comment:
Try this:
```suggestion
* You see that the number of old replicas (adding the replica count from
  `nginx-deployment-1564180365` and `nginx-deployment-2035384211`) is 2, and the number of
  new replicas (from `nginx-deployment-3066724191`) is 1.
```

Reply:
I still believe 3 is the number. The immediate following section reads:

> The output is similar to this:
> 
> ```
> NAME                          DESIRED   CURRENT   READY   AGE
> nginx-deployment-1564180365   3         3         3       25s
> nginx-deployment-2035384211   0         0         0       36s
> nginx-deployment-3066724191   1         1         0       6s
> ```

I think the concept intended to be demonstrated in the article here is that: Kubernetes will not delete an old replica until a new replica is running healthily. In the article, the new replica is stuck by `ImagePullBackOff`.

Reply:
@sftim I still believe 3 is the number. The immediate following section reads:

> The output is similar to this:
> 
> ```
> NAME                          DESIRED   CURRENT   READY   AGE
> nginx-deployment-1564180365   3         3         3       25s
> nginx-deployment-2035384211   0         0         0       36s
> nginx-deployment-3066724191   1         1         0       6s
> ```

I think the concept intended to be demonstrated in the article here is that: Kubernetes will not delete an old replica until a new replica is running healthily. In the article, the new replica is stuck by `ImagePullBackOff`.

Reply:
You're right. I'd meant to adjust the number. I edited my suggestion.

Reply:
@sftim committed as suggested. Thank you very much for your suggestion.